rpc/backend/tx_info.go

```diff
@@ -305,20 +476,21 @@ func (b *Backend) GetTransactionByBlockNumberAndIndex(blockNum rpctypes.BlockNum
 // GetTxByEthHash uses `/tx_query` to find transaction by ethereum tx hash
 // TODO: Don't need to convert once hashing is fixed on Tendermint
 // https://github.com/tendermint/tendermint/issues/6539
-func (b *Backend) GetTxByEthHash(hash common.Hash) (*ethermint.TxResult, error) {
+func (b *Backend) GetTxByEthHash(hash common.Hash) (*ethermint.TxResult, bool, error) {
 	if b.indexer != nil {
-		return b.indexer.GetByTxHash(hash)
+		res, err := b.indexer.GetByTxHash(hash)
+		return res, false, err
```

Comment:
isn't this going to fail for x-chain transactions, since this indexer method only works for standard eth rpc txns?

Reply:
Have tested `eth_getTransactionByHash` and it works, don't think the indexer actually runs on the node